contracts/QuestFactory.sol

```diff
@@ -34,7 +32,7 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
     address public erc1155QuestAddress;
     mapping(string => Quest) public quests;
     RabbitHoleReceipt public rabbitHoleReceiptContract;
-    RabbitHoleTickets public rabbitHoleTicketsContract;
+    address public rabbitHoleTicketsContract;
```

Comment:
Wdym by upgrade safe? Assuming that we get rid of tickets and can overwrite the slot here so that we can use a different address for whatever purpose?